requirements.txt

```diff
@@ -1,3 +1,4 @@
 anymarkup
 lockfile
 jsonpointer
+requests
```

Comment:
@cdrage  this should be enough no? Dockerfiles uses this list for generating package names https://github.com/projectatomic/atomicapp/blob/master/Dockerfiles.git/Dockerfile.centos#L22

@dustymabe  it sould be everywhere i check CentOs RHEL and Fedora and it is there.